.github/styles/Nuclio/ignore.txt

```diff
@@ -0,0 +1,73 @@
+dev
+kubeconfig
+MLRun
+Kubeflow
+serverless
+Katacode
+Serverless
+config
+Minikube
+runtime's
+HTTP
+http
+kubernetes
+stdin
+Gradle
+npm
+Webadmin
+ACK
+NACK
+Kinesis
+retryable
+boolean
+bool
+rebalancing
+rebalance
+OAuth
+Kerberos
+sharding
+sharded
+deployer
+multithreading
+GitHub
+Sarama
+kubectl
+asyncio
+deserialization
+Untags
+untags
+toolchain
+Grafana
+yaml
+backoff
+liveness
+goroutine
+HTTPie
+ns
```

Comment:
aka namespace?
I don't think we should ignore it

Reply:
@TomerShor it was aka nanoseconds. I've removed it from `ignore.txt`.